.gitattributes

```diff
@@ -0,0 +1,2 @@
+# mute json line ending changes
+*.json -crlf
```

Comment:
This can be done using a git config and is platform specific. Please revert this change.

Reply:
then I would like to know how do I mute this locally for one repo without using global config - I see no other option
also this only means to not check (and convert) line endings in json files - I don't find that a big deal